lib/api/controllers/adminController.js

```diff
@@ -119,11 +120,34 @@ function AdminController(kuzzle) {
         modifiedRequestObject = newRequestObject;
 
         _.forEach(kuzzle.pluginsManager.plugins, (object, id) => {
-          pluginsConfig[id] = _.assignIn({}, object);
-          if(pluginsConfig[id].object.context && pluginsConfig[id].object.context.config) {
+          pluginsConfig[id] = JSON.parse(stringifySafe(object));
```

Comment:
If I understand correctly, the returned value will be a dump of the complete plugin. Shouldn't we rather return the configuration stored in `%kuzzle/plugins`?